app/models/user.rb

```diff
@@ -468,6 +468,20 @@ def seed_aspects
     aq
   end
 
+  def send_welcome_message
```

Comment:
Assignment Branch Condition size for send_welcome_message is too high. [24.37/15]